src/utils/index.js

```diff
@@ -72,7 +72,9 @@ function getFieldFromState(state, model) {
 
   if (!form) return null;
 
-  return getField(form, toPath(model).slice(1));
+  const formPath = toPath(form.model);
+  const fieldPath = toPath(model).slice(formPath.length);
```

Comment:
This line changed the output from this function from what it used to be. I was able to fix it using this:

const fieldPath = toPath(model).slice(formPath.length - 1);

Please take another look when you can. Thanks!